sources/packages/backend/workflow/test/test-utils/factories/assessment-consolidated-data.ts

```diff
@@ -203,6 +234,7 @@ export function createFakeConsolidatedFulltimeData(
   const [, programEndYear] = programYear.split("-");
   const assessmentConsolidatedData =
     createFakeAssessmentConsolidatedData(programYear);
+  populateDefaultProgramYearTotals(assessmentConsolidatedData);
```

Comment:
Is there a reason for calling `createFakeAssessmentConsolidatedData` individually for full-time/part-time instead of directly inside the `createFakeAssessmentConsolidatedData`?
If not, can we call ii once from the `createFakeAssessmentConsolidatedData`?